### PR TITLE
Remove and comment out code related to banner

### PR DESCRIPTION
**Pull request by meleongg**
## Description

- removed/commented out banner code
- also added Oliver's link on footer

### CODE REVIEW

File: components/Hero.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const HeroContainer = styled.div`
   padding-bottom: 69%;
   background: url(/assets/hero_illustration.svg);
   background-repeat: no-repeat;
-  background-position: center;
+  background-position: top;
   background-size: contain;
   padding-top: 134px;
```
Comment (sidyakinian) [inline components/Hero.js:18]: Would recommend removing padding-top as well. This used added to shift down the hero, but now without the banner it's shifted down too much (e. g. nwPlus text is a bit low and might be way too low on smaller screens)

File: components/NavBar.js [hunk 2]
```diff
@@ -233,25 +232,22 @@ const NavBar = () => {
   };
 
   const handleScroll = () => {
-    let lastScroll = 0
+    var lastScroll = 0;
```
Comment (sidyakinian) [inline components/NavBar.js:235]: Just wanted to double-check if var is really needed here – seems like `lastScroll` is only used within scope. Correct me if I'm wrong though!

**Review by sidyakinian — CHANGES_REQUESTED** [review]
Left a couple comments

**Review by sidyakinian — APPROVED** [review]

File: components/Hero.js (hunk 1 above)
Comment (meleongg) [reply components/Hero.js:18]: Good catch, I didn't notice that

File: components/NavBar.js (hunk 2 above)
Comment (meleongg) [reply components/NavBar.js:235]: var is outdated - I'll change it to let

**Pull request merged**